src/compiler/create-compiler.js

```diff
@@ -25,10 +25,7 @@ export function createCompilerCreator (baseCompile: Function): Function {
         }
         // merge custom directives
         if (options.directives) {
-          finalOptions.directives = extend(
-            Object.create(baseOptions.directives || null),
-            options.directives
-          )
+          finalOptions.directives = extend({}, finalOptions.directives, options.directives)
```

Comment:
I think that's one of the functions of tests, and since nothing is breaking as we can see from CI, I don't think it matters much. Unless truly, there's a weird reason for doing it the way they did. I'll be happy to know your other thoughts

Reply:
@hirokiosame What's the difference?